Description: Disable the MAGICKCORE_HAVE_POPEN
  This removes support for the pipe (|) prefix for
  reading/writing from/to a shell command.
Origin: http://www.openwall.com/lists/oss-security/2016/05/29/7
Bug-CVE: CVE-2016-5118
Bug-Debian: https://bugs.debian.org/825799
Last-Update: 2016-05-31
---
Index: imagemagick-6.6.9.7/magick/blob.c
===================================================================
--- imagemagick-6.6.9.7.orig/magick/blob.c	2016-06-01 13:29:09.675798317 -0400
+++ imagemagick-6.6.9.7/magick/blob.c	2016-06-01 13:29:09.671798264 -0400
@@ -89,6 +89,9 @@
 #define _O_BINARY O_BINARY
 #endif
 
+
+#undef MAGICKCORE_HAVE_POPEN // CVE-2016-5118
+
 /*
   Typedef declarations.
 */
